miniwin/src/ddraw/ddraw.cpp

```diff
@@ -357,7 +357,7 @@ HRESULT DirectDrawImpl::CreateDevice(
 		DDRenderer = new Citro3DRenderer(DDSDesc.dwWidth, DDSDesc.dwHeight);
 	}
 #endif
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(WINDOWS_STORE)
```

Comment:
I'm assuming that we won't be distributing UWP on desktop computers, that would be my only concern here if we were. Otherwise LGTM

Reply:
yeah UWP is not for Windows desktop